OverlayPlugin.Core/resources/opcodes.jsonc

```diff
@@ -380,5 +380,21 @@
             "opcode": 142,
             "size": 1080
         }
+    },
+    "2023.06.14.0000.0000": {
+        // Region: Global
+        // Version: 6.41
+        "MapEffect": {
+            "opcode": 629,
+            "size": 11
+        },
+        "CEDirector": {
+            "opcode": 777,
+            "size": 16
+        },
+        "RSVData": {
+            "opcode": 115,
+            "size": 1080
+        }
```

Comment:
Is this backwards? Are these the KR opcodes and you didn't mean to change the global ones?

Reply:
It's obviously my mistake. Maybe I dozed off late at night.